canonical/snapcraft.io
# Redirect to login beta

## Done
- Add decorator to model service routes
- If User is not candid authenticated, it redirects to `/login-beta` for authentication and takes user back to the request page
## How to QA
- Go to https://snapcraft-io-4355.demos.haus/login
- Then try accessing any of the model endpoints e.g
    - https://snapcraft-io-4355.demos.haus/admin/store/njwQYXFnS7ppo7LaGxoh7aqVZc1CPi26/models
    - https://snapcraft-io-4355.demos.haus/admin/store/njwQYXFnS7ppo7LaGxoh7aqVZc1CPi26/models/alimot_test_model/policies
- It is expected to prompt you to authenticate again(candid)
- It should display the requested page after authentication
## Issue / Card https://warthogs.atlassian.net/browse/WD-5284
Fixes #

## Screenshots
